src/assets/scripts/client/airline/AirlineController.js

```diff
@@ -10,11 +11,25 @@ const airline = {};
 export default class AirlineController {
     /**
      * @constructor
+     * @for AirlineController
+     * @param airlineList {array}
      */
-    constructor() {
+    constructor(airlineList) {
+        this.airlineCollection = new AirlineCollection(airlineList);
+
         this.airline = airline;
         this.airline.airlines = {};
```

Comment:
Are these two above lines destined for removal I assume? The collection should be able to do whatever these do.